web/src/ui_init.js

```diff
@@ -632,6 +632,7 @@ export function initialize_everything() {
     user_group_edit.initialize();
     stream_edit_subscribers.initialize();
     stream_data.initialize(stream_data_params);
+    compose_ui.initialize_compose_stream_dropdown();
```

Comment:
This should probably become `compose_recipient.initialize()` -- our convention is for all of these functions called from this module to be called `initialize`. (It makes it easier to understand modules without every looking at this one).

I also suspect this should be further down -- maybe next to `compose_pm_pill`, where we're initializing the most similar existing UI component?

Reply:
renamed to `initialize()` and moved next to `compose_pm_pill`